doc/DOWNLOADS.md

```diff
@@ -18,77 +20,53 @@ sudo apt install nicotine
 
 If you prefer to install a .deb package directly, you can download one [here](http://ppa.launchpad.net/nicotine-team/stable/ubuntu/pool/main/n/nicotine/). Unlike the repository installation method, Nicotine+ will not update automatically; you need to download a .deb package each time a new release is available.
 
-### Ubuntu/Debian (Unstable)
+### Fedora
 
-The project builds [daily unstable snapshots](https://code.launchpad.net/~nicotine-team/+recipe/nicotine+-daily) in a separate [unstable PPA](https://code.launchpad.net/~nicotine-team/+archive/ubuntu/unstable). To use it, run the following:
+To install Nicotine+ on Fedora, run the following:
 
 ```sh
-sudo apt install software-properties-common
-sudo add-apt-repository ppa:nicotine-team/unstable
-sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 6CEB6050A30E5769
-sudo apt update
-sudo apt install nicotine
+sudo dnf install nicotine+
 ```
 
-If you prefer to install a .deb package directly, you can download one [here](http://ppa.launchpad.net/nicotine-team/unstable/ubuntu/pool/main/n/nicotine/). Unlike the repository installation method, Nicotine+ will not update automatically; you need to download a .deb package each time a new release is available.
+### Flatpak
+
+If your distribution supports Flatpak, you can install Nicotine+ from Flathub.
+
+- [Download Nicotine+ on Flathub](https://flathub.org/apps/details/org.nicotine_plus.Nicotine)
 
-### Arch Linux/Manjaro/Parabola (Stable)
+### Arch Linux/Manjaro/Parabola
 
 Nicotine+ is available in the community repository of Arch Linux, Manjaro and Parabola. To install, run the following:
 
 ```sh
 sudo pacman -S nicotine+
 ```
 
-### Void Linux (Stable)
+### Void Linux
 
 To install Nicotine+ on Void Linux, run the following:
 
 ```sh
 sudo xbps-install -S nicotine+
 ```
 
```

Comment:
### FreeBSD

To install Nicotine+ on FreeBSD, run the following:

```sh
pkg install py-nicotine-plus
```

Reply:
It was added in https://github.com/nicotine-plus/nicotine-plus/commit/29d9fdb9148ca536985bf018e4dbc434f5cd54f5